### PR TITLE
Add Wiki document on how to generate a sample project on development

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/nimblehq/elixir-templates/issues/157

## What happened

Add Wiki document on how to generate a sample project on development.

## Insight

- Add `Installation.md`

## Proof Of Work

https://github.com/nimblehq/elixir-templates/pull/171/files

### CODE REVIEW

File: .github/wiki/_Sidebar.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Table of Contents
 
 - [[Home]]
+- [[Installation]]
```
Comment (andyduong1920) [inline .github/wiki/_Sidebar.md:4]: the `Installation` term might confuse with the install the template itself (https://github.com/nimblehq/elixir-templates#installation) in a new project part, should we find another term? something like `generate a sample project`

How about? `Test the apply action on development`

File: .github/wiki/Installation.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`mix do deps.get, deps.compile`
+
+7. Choose which template you want to use inside your project
+
+  - `mix nimble_template.gen --api`
+  - `mix nimble_template.gen --live`
+  - `mix nimble_template.gen --web`
+
+8. Answer the interactive prompt that will be displayed during the setup
```
Comment (andyduong1920) [inline .github/wiki/Installation.md:27]: We could show another solution (second solution) by using the `Makefile`

1. Clone [Elixir templates](https://github.com/nimblehq/elixir-templates) project to your local directory.
2. Generate a new Phoenix project 

- Web project: make create_phoenix_project PROJECT_DIRECTORY=sample_project OPTIONS="--no-live"
- Live project: make create_phoenix_project PROJECT_DIRECTORY=sample_project OPTIONS=""
- API project: make create_phoenix_project PROJECT_DIRECTORY=sample_project OPTIONS="--no-html --no-assets"

3. Answer the prompt `Fetch and install dependencies? [Yn]` with `n` (not install)
4. Apply the template
- Web variant: make apply_phoenix_template PROJECT_DIRECTORY=sample_project VARIANT=web
- Live variant: make apply_phoenix_template PROJECT_DIRECTORY=sample_project VARIANT=live
- API variant: make apply_phoenix_template PROJECT_DIRECTORY=sample_project VARIANT=api

File: .github/wiki/_Sidebar.md (hunk 1 above)
Comment (topnimble) [reply .github/wiki/_Sidebar.md]: Fixed in ba44478

File: .github/wiki/Installation.md (hunk 2 above)
Comment (topnimble) [reply .github/wiki/Installation.md]: Fixed in ba44478

File: .github/wiki/Test the apply action on development.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Test the apply action on development
```
Comment (andyduong1920) [inline .github/wiki/Test the apply action on development.md:1]: a small concern, should we name this file as 
`Test-The-Apply-Action-On-Development.md`

File: .github/wiki/_Sidebar.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ## Table of Contents
 
 - [[Home]]
+- [[Test the apply action on development]]
```
Comment (andyduong1920) [inline .github/wiki/_Sidebar.md:4]: How about `[[Test The Apply Action On Development]]`

**Review by andyduong1920 — APPROVED** [review]
just small comment about the naming @topnimble 👍

File: .github/wiki/Test the apply action on development.md (hunk 3 above)
Comment (topnimble) [reply .github/wiki/Test the apply action on development.md]: Fixed in ddc985a

File: .github/wiki/_Sidebar.md (hunk 4 above)
Comment (topnimble) [reply .github/wiki/_Sidebar.md]: Fixed in ddc985a

File: .github/wiki/Test-The-Apply-Action-On-Development.md.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+# Test the apply action on development
```
Comment (andyduong1920) [inline .github/wiki/Test-The-Apply-Action-On-Development.md.md:1]: Pasted_Image_10_03_2022__10_51

Comment (topnimble) [reply .github/wiki/Test-The-Apply-Action-On-Development.md.md]: Fixed in f6e095a

File: .github/wiki/Test-The-Apply-Action-On-Development.md.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+## Manually
```
Comment (longnd) [inline .github/wiki/Test-The-Apply-Action-On-Development.md.md:1]: this file name has a redundant extension `.md.md`

File: .github/wiki/Test-The-Apply-Action-On-Development.md.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+1. Clone [Elixir templates](https://github.com/nimblehq/elixir-templates) project to your local directory.
+
+2. Create a new project using `mix phx.new sample_project` or ``mix new sample_project`` command
```
Comment (longnd) [inline .github/wiki/Test-The-Apply-Action-On-Development.md.md:5]:
```suggestion
2. Create a new project using `mix phx.new sample_project` or `mix new sample_project` command
```

File: .github/wiki/Test-The-Apply-Action-On-Development.md.md (hunk 6 above)
Comment (longnd) [reply .github/wiki/Test-The-Apply-Action-On-Development.md.md:1]: btw, I think the name of this wiki page isn't descriptive
> Test the apply action on development

What is it about? As the content focuses on generating a new phoenix project on the local development environment using the elixir template, should we change the name to reflex that?

File: .github/wiki/Test-The-Apply-Action-On-Development.md.md (hunk 7 above)
Comment (topnimble) [reply .github/wiki/Test-The-Apply-Action-On-Development.md.md]: Fixed in cdc8c5f

File: .github/wiki/Test-The-Apply-Action-On-Development.md.md (hunk 6 above)
Comment (topnimble) [reply .github/wiki/Test-The-Apply-Action-On-Development.md.md]: Fixed in bc86934 and 46d4171

File: .github/wiki/_Sidebar.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ## Table of Contents
 
 - [[Home]]
+- [[Generation]]
```
Comment (andyduong1920) [inline .github/wiki/_Sidebar.md:4]: Don't forget to update the link on the Contribute page @topnimble 

image

Comment (topnimble) [reply .github/wiki/_Sidebar.md:4]: Fixed in d78d0f0

**Review by longnd — APPROVED** [review]

**Pull request merged**